### DISCUSSION TITLE
using transformers as table titles to insert information into SQL databases

### DISCUSSION THREAD

**Discussion by bkojin90** [nextgenhealthcare/connect#5772] [Q&A]
Hi all,

Is it possible in mirth to use transformer output (e.g. ${tablename}) as the table name for inserting into an SQL database?

Where ${tablename} is say, 'table_title'

e.g.

INSERT INTO ${tablename} (column1)
VALUES ('dataexample')

Currently this seems to give an error. 

However it works perfectly if I put:
INSERT INTO table_title (column1)
VALUES ('dataexample')

I would love to be able to do the former if possible in order to utilise dynamic table titles.

Any helped greatly appreciated! :)

**Comment by pacmano1** [comment]
Yes, you can do that.  Your syntax may be vary on what mechanism you use to make database calls. 

Please use code formatting here on Github.   See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

Your terminology needs clarification, what do you mean by transformer output?  

Post screenshots. 

Please refer to the beginner reading material I mentioned on your last post, it really explains a lot of this stuff including understanding when you can and cannot use velocity syntax in assorted places in Mirth.
